napari/components/viewer_model.py

```diff
@@ -219,6 +225,15 @@ def __init__(
         )
         self.__config__.extra = Extra.ignore
 
+        self._dispatch_ctx = Context(
+            self._ctx, self.layers._ctx, self.layers._selection_ctx
+        )
+        # FIXME: ideally the dispatcher should live in the app and then switch contexts
```

Comment:
@aganders3 Could this matter for multicanvas?

Reply:
No sure I understand the _context_ here (ignore me in such case :P), but this is definitely not the plan for the long term: each "canvas" (or "viewer", or whatever we end up calling it) will have its own layerlist.

Reply:
Either way I guess that's a discussion for the multicanvas NAP PR I'm about to open 😄.